.github/workflows/e2e-tests.yaml

```diff
@@ -68,7 +71,7 @@ jobs:
         kubectl apply -f https://raw.githubusercontent.com/metallb/metallb/v0.12.1/manifests/metallb.yaml
         kubectl -n metallb-system wait --timeout=1m --for=condition=Available deployments.apps/controller
 
-        cat <<'EOM' | kubectl create -f-
+        kubectl create -f- <<'EOM'
```

Comment:
Shouldn't there be a space: `-f -` vs `-f-`?

Reply:
Both are working for short flags (single dash versions) but I'll add a space if that looks ambiguous.

Reply:
Addressed in [4edd5801..5d40b9c2](https://github.com/triggermesh/triggermesh/compare/4edd580109b4f6f20dcce598819e3b5a23f9be6b..5d40b9c2eba88353589183cf6b8fd1be281db44c)